java/org/brotli/wrapper/enc/Encoder.java

```diff
@@ -22,18 +23,76 @@
   final ByteBuffer inputBuffer;
   boolean closed;
 
+  /**
+   * https://www.brotli.org/encode.html#aa6f
+   * See encode.h, typedef enum BrotliEncoderMode
+   */
+  public enum Mode {
+    /**
+     * Default compression mode.
+     * In this mode compressor does not know anything in advance about the properties of the input.
+     */
+    GENERIC(0),
```

Comment:
Why simply not use ordinal?

Reply:
The value should be the same as in the C code.
I think it is more clear this way. Otherwise if ordinal is used someone may shuffle/sort the modes later and break it

Reply:
@eustas  Should I rework it to use ordinal ?

Reply:
Yes, please. I guess, the modes won't ever change. For 100% guarantees we could add a unit-test to check that enums have  ordinals that correspond to their C counterparts (hardcoded).